Makefile

```diff
@@ -0,0 +1,34 @@
+ARGS =
+DISTRIBUTION ?= ubuntu
+
+ifeq ($(DISTRIBUTION), ubuntu)
+JSON_FILE = cilium-ubuntu.json
+BOX_FILE = cilium-ginkgo-virtualbox-iso.box
+else ifeq ($(DISTRIBUTION), opensuse)
+# openSUSE Tumbleweed is a rolling release distribution which delivers ISO
+# images daily. It means that the SHA256 checksum of the ISO changes every day
+# and we need to fetch it during box building.
+JSON_FILE = cilium-opensuse.json
+BOX_FILE = cilium-ginkgo-opensuse-virtualbox-iso.box
+ISO_URL = http://widehat.opensuse.org/tumbleweed/iso/openSUSE-Tumbleweed-NET-x86_64-Current.iso
+ISO_CHECKSUM = $(shell $(CURDIR)/tools/opensuse-tumbleweed-checksum.sh)
+ARGS += -var iso_url=$(ISO_URL) -var iso_checksum=$(ISO_CHECKSUM)
+else
+$(error "Distribution $(DISTRIBUTION) is unsupported")
+endif
+
+all: build
+
+build: clean validate
+	packer build $(ARGS) $(JSON_FILE)
+
+validate:
+	packer validate $(JSON_FILE)
+
+clean:
+	rm -Rf output-* *.box tmp packer_cache packer-*
```

Comment:
Hey @mrostecki  could you delete the specified box here please? Opensuse or ubuntu please?

Regards

Reply:
But why? Cannot we just remove everything which may be produced by building, regardless of the distro (that's why I'm removing `*.box` and `packer-*`)? Removing every possibly produced file is what people usually expect from `clean` target.

Reply:
Deciding which file should be cleaned, based on the distro, sounds like a little overengineering for me.

Reply:
In the near future, updating the images to Vagrant cloud will be automatic. If you run the process and you're updating the box file for other distribution can be a race condition.